Source/monster.cpp

```diff
@@ -4188,16 +4188,16 @@ void GolumAi(int i)
 		return;
 	}
 
-	if (golem._mmode == MM_DEATH
-	    || golem._mmode == MM_SPSTAND
-	    || (golem._mmode >= MM_WALK && golem._mmode <= MM_WALK3)) {
+	if (golem._mmode == MonsterMode::Death
+	    || golem._mmode == MonsterMode::SpecialStand
+	    || (golem._mmode >= MonsterMode::MoveNorthwards && golem._mmode <= MonsterMode::MoveSideways)) {

```

Comment:
```suggestion
	if (IsAnyOf(golem._mmode, MonsterMode::Death, MonsterMode::SpecialStand, MonsterMode::MoveNorthwards, MonsterMode::MoveSouthwards, MonsterMode::MoveSideways)) {
```

Reply:
Second half is required:
```suggestion
	if (golem._mmode == MonsterMode::Death
	    || golem._mmode == MonsterMode::SpecialStand
	    || IsAnyOf(golem._mmode, MonsterMode::MoveNorthwards, MonsterMode::MoveSouthwards, MonsterMode::MoveSideways)) {
```